platform/queue/queue.go

```diff
@@ -318,6 +322,58 @@ func (db *Store) pollCommands(pubsub pubsub.PublishSubscriber) error {
 	return nil
 }
 
+func (db *Store) pollRawCommands(pubsub pubsub.PublishSubscriber) error {
```

Comment:
any reason not to re-use the existing command poller (and event, pubsub topic, etc)?

Reply:
I considered sharing the poller, but ultimately thought keeping them separate made it a bit cleaner. If you would rather have them combined in one select (and use one less goroutine), I can make that change easily.

Using a separate event type and pubsub topic allows things to stay *much* simpler. To use the existing `Event` type, we'd have to add a `RawPayload []byte` field, since it doesn't contain a `[]byte` field. Then we'd have to add more logic to the marshal/unmarshal code to differentiate between the two, probably by adding another field (e.g. `type`) to the underlying event protobuf.

By having separate event types, we can keep the same underlying protobuf untouched (since it *does* use a plain []byte field), and differentiate between the two by using two separate topics.

Another thing to consider is that the existing event/queue is real-world tested for several years. While I think the raw command code is solid, if there is some bug, keeping it segregated in another type and topic makes sure it doesn't need to change the existing, working system.

Reply:
Yep you're 100% correct here. I completely forgot we not only marshal/unmarshal command JSON, but in fact marshal it into protobuf. And the current pubsub event is dedicated to that marshaling. The sepratet event and poller make perfect sense in that context. I just forgot about it, lol. :)